docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -56,7 +56,16 @@ We have spent a considerate amount of time thinking about this issue and decided
 Most of our product operators will not set any specific certificate lifetime, so the default applies.
 In case an operator sets a higher lifetime, a tracking issue must be created to document and track the steps to reduce the certificate lifetime.
 
-Users can use podOverrides to extend the certificate lifetime by adding volume annotations. We might add native support to customize certificate lifetimes in the future by using the product CRDs.
+Users can use podOverrides to extend the certificate lifetime by adding volume annotations. We might add native support for customizing certificate lifetimes in the future to the Stacklet CRDs.
+
+==== Certificate Authority rotation
```

Comment:
What about cleanup of expired certificates? I think purging all expired certs (only for autoGenerate mode) would be easy enough to include, WDYT?

Reply:
It would be easy enough to add, it's mostly a balancing act of "what happens if we delete a CA too early?".

We could add it behind a flag though.

Reply:
I mean is there any reason to keep an expired ca? Or to put it differently: Can we safely assume none of our TLS clients ignore cert lifetimes? I would tend towards it's safe to assume

Reply:
Ok for me. It would be great if we could add a sentence to the docs similar to "Currently expired CA certificates are not removed and need to be cleaned up manually"

Reply:
edf2bb52f5673233c680383f7bc3f6449507f375